s3transfer/manager.py

```diff
@@ -479,15 +479,19 @@ def _register_handlers(self):
     def __enter__(self):
         return self
 
-    def __exit__(self, exc_type, *args):
+    def __exit__(self, exc_type, exc_value, *args):
         cancel = False
+        cancel_msg = ''
         # If a exception was raised in the context handler, signal to cancel
         # all of the inprogress futures in the shutdown.
         if exc_type:
             cancel = True
-        self.shutdown(cancel)
+            cancel_msg = exc_value
+            if exc_type == KeyboardInterrupt:
+                cancel_msg = 'keyboard interrupt'
```

Comment:
What do you think about using `repr` here when there's no message to be more generic?